doc/docs-requirements.txt

```diff
@@ -1,2 +1,5 @@
 tornado
 aiohttp
+Sphinx~=4.1
+sphinx_rtd_theme~=0.5
```

Comment:
Are these not implicitly specified by installing `Sphinx`? Why do they need to be explicitly specified now?

Reply:
They are not. These are readthedocs specific dependencies. The docs recommend pinning them: https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html#pinning-dependencies